ContestUtil/src/org/icpc/tools/contest/util/ImagesGenerator.java

```diff
@@ -187,6 +199,57 @@ public void generateTeamPhotos() {
 		generateImages("teams", "photo", "jpg", spec);
 	}
 
+	public static int shrinkDirectionRight(int left, int w, int h, int[] pix, boolean horizontal) throws Exception {
```

Comment:
These are more start/end than right/left since they work in two directions?

Reply:
Yes, that's a better name.